### ISSUE TITLE
Instruction range is zero if the instruction's keyword is only of length 1

### ISSUE THREAD

**Issue by rcjsuen** [rcjsuen/dockerfile-ast#1]
```Dockerfile
F alpine
```
Given the above Dockerfile where the instruction is only of length 1, an incorrect range of length 0 will be returned from `getInstructionRange()`.

```
{"start":{"line":0,"character":0},"end":{"line":0,"character":0}}
```

**Comment by rcjsuen** [comment]
Whoops. Closed this with the commit message in f6b603b8d6e37b0dec2a6c2f8659e0cea8ab079f when I should have been referring to issue #2.

**Comment by rcjsuen** [comment]
Typed in the wrong commit comment again. :( This has been fixed by 4607a8b66ef6641208e855ca3f0dc84cf07b90d5.
